exercises/practice/list-ops/test_list_ops.c

```diff
@@ -1,14 +1,38 @@
-#include "test-framework/unity.h"
+#include <stdarg.h>
 #include "list_ops.h"
+#include "test-framework/unity.h"
 
-const int MAX_STRING_LEN = 100;
+static const int MAX_STRING_LEN = 100;
+static list_t *list = NULL;
+static list_t *list2 = NULL;
+static list_t *actual = NULL;
+static char *error_message = NULL;
 
 void setUp(void)
 {
 }
 
+static void release(int count, ...)
+{
+   va_list ptr;
+   va_start(ptr, count);
+   for (int i = 0; i < count; i++) {
```

Comment:
```suggestion
   for (size_t i = 0; i < count; i++) {
```

Reply:
FYI also, that `++i` rather than `i++` would be ever so slightly more efficient.